romanesco/plugins/gaia/classes.py

```diff
@@ -0,0 +1,160 @@
+import json
+import sys
+import ast
+import os
+from romanesco.format import validators
+
+
+class RomanescoTypeConverter(object):
+    def __new__(cls, value, **kwargs):
+
+        if cls.is_valid(value, **kwargs):
+            return cls.convert(value, **kwargs)
+        else:
+            raise Exception("Invalid Romanesco Type")
+
+    @classmethod
+    def is_valid(cls, value, **kwargs):
+        # Currently we don't actually validate anything
+        return True
+
+    @classmethod
+    def convert(cls, value, **kwargs):
+        # Currently we don't actually convert anything
+        # Would have to resolve the format of value before
+        # converting it to the format in kwargs
+        return value
+
+# Define the Output converter
+Output = type("Output", (RomanescoTypeConverter,), {})
+
+romanesco_types = []
+
+module = sys.modules[__name__]
+
+for key, val in validators.items():
+    romanesco_types.append(key.title())
+
+    if not hasattr(module, key.title()):
+        setattr(module,
+                str(key.title()),
+                type(str(key.title()), (RomanescoTypeConverter, ), {}))
```

Comment:
@zachmullen @jbeezley moving ```romanesco_types``` into ```RomanescoTypeConverter``` makes sense to me. 

The goal of putting those classes into ```module.__dict__``` was to make them directly importable from the module allowing end users to wrap injected variables in things like ```Table()```  or ```String()```   or ```Graph()``` classes.  Unless i'm missing something-  If we push the classes inside ```RomanescoTypeConverter```  then it becomes more complicated to wrap injected variables (e.g.,  ```RomanescoTypeCoverter.romanesco_type["string"]()```) right?

My hope was this would be a half measure between a full OOP refactor of the Type/Format system and that eventually the metaclass stuff could be cleared out of this plugin -  or is that not in the cards for romanesco?